### ISSUE TITLE
Standardize log-in screen.

### ISSUE THREAD

**Issue by orbitmechanic** [orbitmechanic/USAMobile#331]
The log-in screen will:
- no longer be a side-drawer.  
- no longer offer the "Use MetaMask" option.
-now be a full screen with components centrally aligned.
-offer "user name" and "password" fields.
-Have a "reset password" button.

**Comment by orbitmechanic** [comment]
![image](https://user-images.githubusercontent.com/56613757/148086230-04e28dc6-6b41-4a2b-9224-cf00991e4807.png)
That's it.

**Comment by orbitmechanic** [comment]
I should probably also mention that I had Nadeem do this a while ago for the purpose of integrating with the marketing website.  There should already be a login and a signup route.  We just need to move them around a bit so they're the front page?

**Comment by orbitmechanic** [comment]
Looks great!  Somehow the middle section came out in black and it's a bit illegible against the dark blue.
<img width="359" alt="Screen Shot 2022-01-05 at 5 39 25 PM" src="https://user-images.githubusercontent.com/56613757/148299819-05ebdb62-1ed3-4112-880f-2db9ba536140.png">

And loose the user name field.

**Comment by orbitmechanic** [comment]
Log-in screen needs to be e-mail and password only.

**Comment by orbitmechanic** [comment]
The good:

- White text is restored!
- I was able to get through this thing by editing my credentials server-side. But in doing so I made some realizations:

The "needs a bit more work":
- "Username" is really "email".  Change the default text accordingly.  We no longer have user names.
- Pressing the return button does nothing from the username (really e-mail) field.  And that's ok.
- Pressing the return button from the password field on the log-in screen should execute a press on the log-in button.

![Screen Shot 2022-01-06 at 9 31 04 AM](https://user-images.githubusercontent.com/56613757/148398624-d6c07037-c555-4da3-b952-f616175fa5a3.png)

For the Register page:

- Remove the "User name" field.
- Switch the order.  e-mail should be the first field, followed by password.

![Screen Shot 2022-01-06 at 9 34 53 AM](https://user-images.githubusercontent.com/56613757/148399134-c735a46a-71aa-4ce7-869f-adca8af3d205.png)
